docs/.env.production.example

```diff
@@ -0,0 +1,7 @@
+# An example of the .env file you should have in the root directory (clone root, not system root)
+ 
+# The port to run councilwatch on
+COUNCILWATCH_PORT=8080
+
+# The database password
+POSTGRES_PASSWORD=changeme
```

Comment:
I'll make sure not to change this when I deploy prod. It might just be crazy enough to be secure!